cevo/.htaccess

```diff
@@ -16,9 +16,10 @@ RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
 RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
 RewriteRule ^$ https://shekarpour.github.io/cevo.io/ 
 
-# Rewrite rule to serve RDF/XML content if requested
+# Rewrite rule to serve RDF/XML OWL content if requested
 RewriteCond %{HTTP_ACCEPT} application/rdf\+xml [OR]
 RewriteCond %{HTTP_ACCEPT} application/owl\+xml
+RewriteCond %{HTTP_ACCEPT} ^.owl
```

Comment:
I don't think this will work.  `Accept` can't be both `application/owl+xml` and start with `.owl`.  We can help with the rules if you'd like to explain what you are trying to do.